### ISSUE TITLE
`06-ba-08` changed but not mentioned in the microcode-20240910 release notes

### ISSUE THREAD

**Issue by stormi** [intel/Intel-Linux-Processor-Microcode-Data-Files#83]
The commit shows the file was modified.

**Comment by hmh** [comment]
0x000b06a8 was updated to the same microcode revision as the other steppings (0x000b06a2, 0x0000b06a3) were, so it looks like a simple omission from the release notes.   Still, it would be nice to know for sure whether this update was intended or not [for this release].

**Comment by whpenner** [comment]
Yes, you are correct.  The MCU was updated for 06-ba-08 and is missing from the releasenotes.md file.

**Comment by whpenner** [comment]
Just to check, are you asking about 06-ba-08 because somebody has a processor with this CPUID or because it was included in the updated files?

**Comment by stormi** [comment]
Just because it was included in the updated files.

**Comment by hmh** [comment]
Well, since it is in the release (even if not in the release notes), might as well include it in the packages.  It might come useful to someone, systems vendors have seriously screwed up and sold inventory with pre-gold firmware before, and I certainly don't trust them AT ALL to not ship gold firmware with outdated PI and MCUs inside either.

**Comment by hmh** [comment]
This signature is an extended signature in a single microcode data file that covers sigs 0x000b06a2, 0x000b06a3 as well.   It doesn't even make sense to *not* distribute it, and it could be useful to someone when the first batches of systems with those hit the stores, should some system vendor out there screw it up and ship outdated firmware (it would NOT be the first time it happened).
